feat(docs): add keyboard shortcuts and new guide for missing data alerts (#1869)
SigNoz/signoz.io: data/docs/product-features/keyboard-shortcuts.mdx

---
date: 2025-08-28
title: Keyboard Shortcuts
id: keyboard-shortcuts
---

> SigNoz provides keyboard shortcuts to help you navigate the application more efficiently. This document covers all available shortcuts organized by feature area.

Reviewer: ```suggestion
SigNoz provides keyboard shortcuts to help you navigate faster. Below is a comprehensive list of all available shortcuts organized by features.
```

> **Note:** On Windows and Linux, use `Ctrl` instead of `Cmd` for all keyboard shortcuts listed below.

Reviewer: ```suggestion
> **Note:** On Windows and Linux, use `Ctrl` instead of `Cmd` for all keyboard shortcuts listed below.
```

Can create an info admonition if we want to show this note more prominently.

Author: No we don't want to show it too prominently so let's keep it same

## Global Shortcuts

These shortcuts work from anywhere within the SigNoz application:

| Keyboard Shortcut | Description |
|------------------|-------------|
| `cmd+k` | Open global action search |
| `shift+s` | Navigate to Services List |
| `shift+t` | Navigate to Traces Explorer |
| `shift+l` | Navigate to Logs Explorer |
| `shift+d` | Navigate to Dashboards List |
| `shift+a` | Navigate to Alerts List |
| `shift+e` | Navigate to Exceptions List |
| `shift+m` | Navigate to Messaging Queues |
| `shift+b` | Toggle sidebar visibility |

> | `shift+h` | Navigate to Home |

Reviewer: ```suggestion
| `cmd + K` | Open global action search |
| `shift + S` | Navigate to Services List |
| `shift + T` | Navigate to Traces Explorer |
| `shift + L` | Navigate to Logs Explorer |
| `shift + D` | Navigate to Dashboards List |
| `shift + A` | Navigate to Alerts List |
| `shift + E` | Navigate to Exceptions List |
| `shift + M` | Navigate to Messaging Queues |
| `shift + B` | Toggle sidebar visibility |
| `shift + H` | Navigate to Home |
```

## Logs Explorer Shortcuts

> These shortcuts are available when using the Logs Explorer:

Reviewer: ```suggestion
These shortcuts are available when using the [Logs Explorer](https://signoz.io/docs/product-features/logs-explorer/):
```

| Keyboard Shortcut | Description |
|------------------|-------------|

> | `cmd+enter` | Stage and Run the current query |
> | `cmd+/` | Toggle all filters in the filters dropdown |

Reviewer: ```suggestion
| `cmd + return` | Stage and Run the current query |
| `cmd + /` | Toggle all filters in the filters dropdown |
```

Reviewer: Can add a note that in windows/linux, `return` will be `Enter`

Author: I think enter is more prominent so will keep it as it is

## Query Builder Shortcuts

> These shortcuts are available when using the Query Builder:

Reviewer: ```suggestion
These shortcuts are available when using the [Query Builder](https://signoz.io/docs/userguide/query-builder/):
```

| Keyboard Shortcut | Description |
|------------------|-------------|

> | `cmd+enter` | Stage and Run the current query |

Reviewer: ```suggestion
| `cmd + return` | Stage and Run the current query |
```

Reviewer: Can add a note that in windows/linux, this will be `Enter`

## Dashboard Shortcuts

> These shortcuts are available when working with dashboards:

Reviewer: ```suggestion
These shortcuts are available when working with [dashboards](https://signoz.io/docs/userguide/manage-dashboards/):
```

Hyperlinking it to Manage page since we don't have a [dashboards get started](https://github.com/SigNoz/signoz-web/issues/1871) page as of now.

| Keyboard Shortcut | Description |
|------------------|-------------|
| `cmd+s` | Save Changes for panel |

> | `cmd+d` | Discard Changes for panel |

Reviewer: ```suggestion
| `cmd + S` | Save Changes for panel |
| `cmd + D` | Discard Changes for panel |
```

## Tips for Using Keyboard Shortcuts

- Keyboard shortcuts can significantly speed up your workflow once you memorize the most commonly used ones
- Use `cmd+K` to open the global action search to quickly navigate or toggle settings
- Start with the global navigation shortcuts (`shift+s`, `shift+t`, `shift+l`, etc.) as these are used most frequently
- The `cmd+enter` shortcut for running queries is particularly useful when building complex queries

> - Use `shift+b` to toggle the sidebar for a cleaner view when focusing on data analysis

Reviewer: ```suggestion
- Use `cmd + K` to open the global action search to quickly navigate or toggle settings
- Start with the global navigation shortcuts (`shift + S`, `shift + T`, `shift + L`, etc.) as these are used most frequently
- The `cmd + return` shortcut for running queries is particularly useful when building complex queries
- Use `shift + B` to toggle the sidebar for a cleaner view when focusing on data analysis
```
